packages/accordions/src/elements/accordion/Accordion.tsx

```diff
@@ -39,11 +39,15 @@ interface IAccordionProps extends Omit<HTMLAttributes<HTMLDivElement>, 'onChange
   isBare?: boolean;
   /** Determines if panels can be collapsed in an uncontrolled accordion */
   isCollapsible?: boolean;
-  /** Reduces padding on accordion headers and panels */
+  /** Reduces the padding on the accordion headers and panels */
   isCompact?: boolean;
-  /** Determines if multiple panels can be expanded at the same time in an uncontrolled accordion */
+  /** Determines if multiple panels can be expanded simultaneously in an uncontrolled accordion*/
```

Comment:
[nit] please retain the trailing space before the comment close.

Reply:
Fixed in latest push